### ISSUE TITLE
#-1745712418------------- Links must have discernible text

### ISSUE THREAD

**Issue by khyatit13** [khyatit13/Coding-practice#1823]
<strong>PageName:</strong> LayersPage  

<strong>Source:</strong>`<a id="edit-button-60c8b32ea6b84717ae7fc44d6cf3b022" href="/layers/60c8b32ea6b84717ae7fc44d6cf3b022/edit" aria-describedby="calcite-tooltip-1d0f75f4-ee54-f3ac-eb87-ed7acb08831d">`

<strong>Element Location: </strong>#edit-button-60c8b32ea6b84717ae7fc44d6cf3b022

<strong>Impact: </strong>serious

<strong>Failure Summary:</strong> Fix all of the following:
  Element is in tab order and does not have accessible text

Fix any of the following:
  Element does not have text that is visible to screen readers
  aria-label attribute does not exist or is empty
  aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
  Element has no title attribute

<strong>Tags:</strong> cat.name-role-value,wcag2a,wcag412,wcag244,section508,section508.22.a,ACT

<strong>HelpUrl:<strong>https://dequeuniversity.com/rules/axe/4.6/link-name?application=axeAPI
<strong>Screenshot:<strong>`C:\Automation\Workspace\Velocity\Screenshot\LayersPage\-1745712418_Links must have discernible text.png`
